.github/workflows/provider.yml

```diff
@@ -0,0 +1,95 @@
+name: Provider
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php-version }} / ${{ matrix.provider }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.3', '7.4', '8.0']
+        provider:
+          - AlgoliaPlaces
+          - ArcGISOnline
+          - AzureMaps
+          - BingMaps
+          - Cache
+          - Chain
+          - FreeGeoIp
+          - GeoIP2
+          # - GeoIPs
+          - GeoPlugin
+          - GeocodeEarth
+          - Geonames
+          - GoogleMaps
+          - GoogleMapsPlaces
+          - GraphHopper
+          - Here
+          - HostIp
+          - IP2Location
+          # - IP2LocationBinary
+          - IpInfo
+          - IpInfoDb
+          - Ipstack
+          - LocationIQ
+          - MapQuest
+          - Mapbox
+          # - Mapzen
+          - MaxMind
+          - MaxMindBinary
+          - Nominatim
+          - OpenCage
+          - OpenRouteService
+          - Pelias
+          - Photon
+          - PickPoint
+          - TomTom
+          - Yandex
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use PHP ${{ matrix.php-version }}
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+        extensions: curl
+    - name: Validate composer.json and composer.lock
+      working-directory: ./src/Provider/${{ matrix.provider }}
+      run: composer validate
+    - name: Install dependencies
+      working-directory: ./src/Provider/${{ matrix.provider }}
+      run: composer install --no-progress
```

Comment:
We don't have composer.lock in git, right? If so, we can use composer update.

Reply:
Indeed, we don't commit the `composer.lock` file.
I'll switch to `composer update`. 👍